### PR TITLE
fix for two phase gas-water system

### PR DESCRIPTION
**Pull request by plgbrts**
Note: two-phase gas-water system is not yet enabled

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp [hunk 1]
```diff
@@ -85,7 +85,7 @@ namespace {
         const auto gas    = ph.active(::Opm::Phase::GAS);
 
         const auto threeP = gas && oil && wat;
-        const auto twoP = (!gas && oil && wat) || (gas && oil && !wat) ;
+        const auto twoP = (!gas && oil && wat) || (gas && oil && !wat) || (gas && !oil && wat) ;
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp:88]: Now that we recognise all two-phase combinations we **might** alternatively wish to express this as
```C++
const auto twoP = (gas + oil + wat) == 2;
```
but it's not clear to me that that's easier to read.

Comment (plgbrts) [reply src/opm/parser/eclipse/EclipseState/Grid/SatfuncPropertyInitializers.cpp:88]: I like the suggestion and I think it is easier to read . Then also use:  const auto threeP = (gas + oil + wat) == 3; ?

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me.  I'll run a build check and merge if green.

**Comment by bska** [comment]
jenkins build this please

**Comment by bska** [comment]
Build check is green.  I'll merge into master.

**Pull request merged**